maintainers/maintainer-list.nix

```diff
@@ -8923,7 +8923,7 @@
     name = "pasqui23";
   };
   patryk27 = {
-    email = "wychowaniec.patryk@gmail.com";
+    email = "pwychowaniec@pm.me";
```

Comment:
A separate commit please? (Same PR is fine.)

Reply:
Sure, will do 🙂 Do we have a standardized format for commit messages related to such changes?

Reply:
I don't know. But git log on the maintainers file will probably give you some good ideas.

Reply:
kk, done 🙂